eventnotifier/http/http.go

```diff
@@ -46,28 +48,48 @@ type Notifier struct {
 	authTokenProvider BearerAuthTokenProvider
 }
 
-type HTTPNotifierEvent struct {
+// NotifierEvent represents a notification event
+type NotifierEvent struct {
```

Comment:
Since this struct is never used outside the http package, could we either keep the http prefix or stop exporting this struct?

Reply:
This struct is used in our internal api to deserialized the body of requests sent by the http notifier of the chaos-controller.